DataModel/Protocol/Protocol-definition.yml

```diff
@@ -0,0 +1,285 @@
+Protocol:
+  description: Describes the protocols followed in a study, including details such
+    as the cell capture system, library protocol, the chemistry kit and sampling technique
+    used, as well as sequencing related information.
+  properties:
+    donor_id:
+      description: Foreign key to map to donor ID
+      type: string
+      format: alphanumeric
+      pattern: '"^DO-[0-9]{4}$"'
+    project_id:
+      description: Foreign key to map a project
+      type: string
+      format: alphanumeric
+      pattern: '"^P[0-9]{3}$"'
+    protocol_id:
+      description: Primary key for the protocol ID
+      type: string
+      format: alphanumeric
+      pattern: '"^PRT[0-9]{3}$"'
+    protocol_name:
+      description: Name given to the protocol such as single cell sequencing protocol,
+        Cytokine staining Protocol
+      type: string
+      format: text
+    protocol_description:
+      description: Description that provides enough information about the protocol
+      type: string
+      format: text
+    protocol_core:
+      description: Core protocol-level information
+      type: string
+      format: text
+    computational_method:
```

Comment:
@ishitasati11 Are we only storing URL here? 
A suggestion would be to store the URL as a subproperty of this attribute in `url`.